Automattic/wp-calypso
# Happychat: Add notification sound for new messages

Plays a sound each time a message is received from the operator.

Tested on:
 - [x] Mac/Chrome 55
 - [x] Mac/Safari 10
 - [x] Mac/Firefox
 - [x] Win/Chrome
 - [x] Win/Firefox

### How to test

1. Check out the `add/happychat/notification-sound` branch
1. Visit http://calypso.localhost:3000/help/contact, logged in as a user with Live Chat enabled
1. Start a chat (You may need to log in to the staging happychat HUD and make yourself available as an operator)
1. Make sure your sound is on :)
1. When the operator sends a message, you should hear a sound
